docs/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
I think the license header is worng here isn't it? It should be Red Hat license header.